releaseNotes/AzureSpringCloudAppConfigurationConfig.md

```diff
@@ -0,0 +1,38 @@
+# azure-spring-cloud-appconfiguration-config
+
+[Source code][source_code] | [Package (Maven)][package] | [README][readme] | [Product documentation][docs] | [Samples][samples]
+
+# azure-spring-cloud-appconfiguration-config-web
+
+[Source code web][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
+
+## 2.0.0-beta.1 - May 5, 2021
+
+* Enables loading of configurations from multiple config stores.
+* Each store can load configurations by using key/label `selects`. Many `selects` can be used, default is key = `/applicaiton/*` with label = `${spring.profiles.active}` or if null `\0`. See [README][readme] for full configuration.
+* Authentication vis Connection String, Managed Identity, or any method supported by Azure Identity SDK, see Token Credential Provider in README.
+* Each individual store can have monitoring enabled and disabled and have a cache-expiration time set.
+  * Monitoring requires at least one trigger to be set. `spring.cloud.azure.appconfiguration.stores[0].monitoring.triggers[0].key` is required `spring.cloud.azure.appconfiguration.stores[0].monitoring.triggers[0].label` is optional.
+  * Monitoring has both Pull and Push model options.
+    * Pull checks the config store for changes based on Activity when the cache has expired.
```

Comment:
Cache never expires. It's the 'refresh' that may expire and when it expires we will check with the store. Even when a refresh fails, we will still use the cache. This causes a lot of confusion to customers that they worry about their app will have no cached configuration to use if a refresh fails.

Reply:
I have re worded this, but still use cache as that is the name of the configuration.